include/arch/arm/aarch32/irq.h

```diff
@@ -235,6 +235,15 @@ extern void _isr_wrapper(void);
 
 #endif /* _ASMLANGUAGE */
 
+#ifdef CONFIG_ARMV6_M_ARMV8_M_BASELINE
+#define NVIC_NODEID DT_INST(0, arm_v6m_nvic)
+#elif CONFIG_ARMV7_M_ARMV8_M_MAINLINE
+#define NVIC_NODEID DT_INST(0, arm_v7m_nvic)
+#elif CONFIG_ARMV8_M_BASELINE || CONFIG_ARMV8_M_MAINLINE
```

Comment:
this is not a proper #elif - the compound defines are also set, so this elif is never entered

Reply:
fixed.